src/renderer/vt/paint.cpp

```diff
@@ -424,10 +424,15 @@ using namespace Microsoft::Console::Types;
     // the inbox telnet client doesn't understand the Erase Character sequence,
     // and it uses xterm-ascii. This ensures that xterm and -256color consumers
     // get the enhancements, and telnet isn't broken.
+    //
+    // GH#13229: ECH and EL don't fill the space with "meta" attributes like

```

Comment:
I mean, don't we still have the "background" version of this problem? `\e[40m                               x                \e[m` doesn't require any extended attributes.

Reply:
more tests you got it